RELEASING.md

```diff
@@ -0,0 +1,23 @@
+# Release procedure
+
+## v0.23.xx releases
+
+1. Check if there's a milestone for the release. Are we in a good state for the release? Are there any outstanding PRs that could be merged or should be waited for?
+1. Press the "Draft a new release" button in GitHub.
+1. Create a new tag with the appropriate name (e.g. v0.23.28); also name the release in GitHub accordingly.
+1. Make sure you're targeting the `series/0.23` branch.
+1. Press the "Generate new release notes" button in GitHub.
+1. Manually edit the generated release notes:
+    - Review the auto-labeling and categorization of PRs.
+    - Put "Behind the scenes" section in to a `details` block.
```

Comment:
I _think_ this could be automated in the template, but it's not yet.

Reply:
I took a look at the docs and I think the category titles and labels (and an author exclusion) is the only configuration we have. I don't see a way to add anything to a particular category body
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuration-options